readme.md

```diff
@@ -233,13 +233,13 @@ Training smaller function-specific models and deploying them is handled by the M
 <!-- TOC --><a name="intro"></a>
 ### Intro
 <!-- TOC --><a name="what-is-monkey-patch-in-plain-words"></a>
-#### What is Monkey-patch in plain words?
-Monkey-patch is a simple and seamless way to create LLM augmented functions in python, which ensure the outputs of the LLMs follow a specific structure. Moreover, the more you call a patched function, the cheaper and faster the execution gets.
+#### What is Monkey Patch in plain words?
+Monkey-Patch is a simple way to create LLM-powered functions in Python, which ensure the outputs of the LLMs follow a specific structure. Instead of large text responses like in GPT-4, these functions behave just like Python function calls, but without the manual coding and testing. Moreover, the more you call a patched function, the cheaper and faster the execution gets. 
 
 <!-- TOC --><a name="how-does-this-compare-to-other-frameworks-like-langchain"></a>
-#### How does this compare to other frameworks like Langchain?
-- **Langchain**: Monkey-Patch has a narrower scope than Langchain. Our mission is to ensure predictable and consistent LLM execution, with automatic reductions in cost and latency through finetuning.
-- **Magentic**: Monkey-Patch offers two main benefits compared to Magentic, namely; lower cost and latency through automatic distillation, and more predictable behaviour through test-driven alignment. Currently, there are two cases where you should use Magentic, namely: where you need support for tools (functions) - a feature that is on our roadmap, and where you need support for asynchronous functions.
+#### How does this compare to other frameworks like LangChain?
+- **Langchain**: Monkey-Patch has a narrower scope than LangChain. Our mission is to ensure predictable and consistent LLM execution, with automatic reductions in cost and latency through finetuning.
+- **Marvin AI**: Monkey-Patch offers two main benefits compared to Magentic, namely; lower cost and latency through automatic distillation, and more predictable behaviour through test-driven alignment. Currently, there are two cases where you should use Magentic, namely: where you need support for tools (functions) - a feature that is on our roadmap, and where you need support for asynchronous functions.
```

Comment:
@dnlkwak you only changed one reference from 'Magentic', not all 3.

Reply:
I'm OK with changing it to "Marvin AI/Magentic" for this. I dunno if it's worth having LangChain, Magnetic, and Marvin AI in separate bullet points. Mostly bc IDK the main differences b/w Marvin AI vs. Magnetic.